framework_tvb/tvb/core/services/import_service.py

```diff
@@ -315,36 +296,6 @@ def _store_imported_images(self, project):
                     self._populate_image(os.path.join(root, file_name), project.id)
 
 
-    def import_project_operations(self, project, import_path, dt_burst_mappings=None, burst_ids_mapping=None):
-        """
-        This method scans provided folder and identify all operations that needs to be imported
-        """
-        op_paths = self._append_tmp_to_folders_containing_operations(import_path)
-        operations = self._load_operations_from_paths(project, op_paths)
-
-        imported_operations = []
-
-        # Here we process each operation found
-        for operation in operations:
-            self.logger.debug("Importing operation " + str(operation))
-            old_operation_folder, _ = os.path.split(operation.import_file)
-            operation_entity, datatype_group = self.__import_operation(operation)
-
-            # Rename operation folder with the ID of the stored operation
-            new_operation_path = FilesHelper().get_operation_folder(project.name, operation_entity.id)
-            if old_operation_folder != new_operation_path:
-                # Delete folder of the new operation, otherwise move will fail
-                shutil.rmtree(new_operation_path)
-                shutil.move(old_operation_folder, new_operation_path)
-
-            operation_datatypes = self._load_datatypes_from_operation_folder(new_operation_path, operation_entity,
-                                                                             datatype_group)
-            self._store_imported_datatypes_in_db(project, operation_datatypes, dt_burst_mappings, burst_ids_mapping)
-            imported_operations.append(operation_entity)
-
-        return imported_operations
-
-
```

Comment:
Again. Put this back, and maybe just hard-code the constant FilesHelper.TVB_OPERARATION_FILE  directly here so you can remove it from FilesHelper

Reply:
Okay, I will add everything I deleted in import_service back.